terraform/gamma1.tfvars

```diff
@@ -0,0 +1,54 @@
+mesh_proxmox_node             = "jon"
+mesh_proxmox_template_image   = "debian-cloud"
+mesh_proxmox_storage_location = "local-lvm"
+mesh_env_name                 = "gamma1"
+mesh_local_user               = "debian"
+mesh_mgr_ips = [
+  "10.70.90.168",
+]
+mesh_mgr_cores = [
+  4,
+]
+mesh_mgr_ram_mb = [
+  4096,
+]
+mesh_mgr_disk_size = [
+  50,
+]
+mesh_ips = [
+  "10.70.90.199",
+  "10.70.90.162",
+]
+mesh_worker_cores = [
+  4,
+  4,
+  4,
+  4,
+]
+mesh_worker_ram_mb = [
+  4096,
+  4096,
+  4096,
+  4096,
+]
+mesh_worker_disk_size = [
+  150,
+  150,
+  150,
+  150,
+]
+mesh_lb_ip        = "10.70.90.197"
+mesh_gateway      = "10.70.90.1"
+mesh_networkrange = "24"
+mesh_net_block    = "10.70.90.0"
+mesh_external_ips = [
+  "199.170.132.42",
+]
+meshdb_fqdn = [
+  "gammadb.nycmesh.net",
+  "adminmap.gammadb.nycmesh.net",
+  "los-backend.gammadb.nycmesh.net",
+  "los.gammadb.nycmesh.net",
+  "forms.gammadb.nycmesh.net",
+  "adminmap.gammadb.nycmesh.net",
```

Comment:
```suggestion
```

Reply:
What if we want to deploy that?

Reply:
It is a dupe

Reply:
good catch lol. This is why we don't leave PRs open for months 🤦🏻